BlocSpot/app/src/main/java/com/izor066/android/blocspot/ui/MainActivity.java

```diff
@@ -122,5 +132,30 @@ public void onPointOfInterestClick(PointOfInterest pointOfInterest) {
         startActivity(intent);
     }
 
+    private void showEditDialog() {
+        android.support.v4.app.FragmentManager fm = getSupportFragmentManager();
```

Comment:
I think there's no need to specify the full package here. `FragmentManager` should be enough. 
As a best practice, we should not use abbreviations. In this case it's not that bad because the scope and lifetime of the variable is pretty short (only in this private method), but if the variable were to be a field, than `fm` wouldn't be so descriptive. I personally prefer `fragmentManager` to `fm` :smile: